### DISCUSSION TITLE
Are vertical reference lines possible?

### DISCUSSION THREAD

**Discussion by kevinbror** [Abhinandan-Kushwaha/react-native-gifted-charts#865] [Q&A] [answered]
I'm building a multi line (with dataSet) chart component and experimenting with reference lines. As you can see in the screenshot it is horizontal. I don't see a way to make one vertical along the y axis. Am I missing that in the docs?

Here is my config:
```
dataSet={data}
{...styles.lineChart}
formatYLabel={yAxisFormatter}
showReferenceLine1={true}
referenceLine1Position={3000000}
referenceLine1Config={{
  thickness: 1,
  color: 'red',
  labelText: 'this is a reference line',
  labelTextStyle: {
    color: 'red',
  },
  type: 'dashed',
}}
```

<img width="419" alt="Screenshot 2024-10-10 at 3 06 20 PM" src="https://github.com/user-attachments/assets/ee5d207a-7892-4ec8-b59b-62bcd224ac9e">

**Comment by kevinbror** [comment]
For clarity this is what I would like to do (this chart build for web with recharts)

<img width="649" alt="Screenshot 2024-10-10 at 3 06 46 PM" src="https://github.com/user-attachments/assets/b7a05f44-894c-4bda-aebf-bf905981cbcb">

**Reply by Abhinandan-Kushwaha** [reply, marked as answer]
In the data array, you can add the property `showVerticalLine: true` inside the object where you want to show the vertical line. See [data item props table](showVerticalLine)

**Reply by kevinbror** [reply]
Ah thanks
